### DISCUSSION TITLE
Do we need prefixes with the scope sm:<w-auto item-start flex-row>

### DISCUSSION THREAD

**Discussion by detoner777** [tailwindlabs/tailwindcss#9126] [Ideas]
What does the community think about, if we add a prefix with the scope (something like this: `sm:<w-auto item-start flex-row>`) to reduce the length of the classes string and get more readability:
instead :
 ` <div className="w-48 h-full flex flex-col  items-center  text-base font-normal sm:w-auto sm:flex-row sm:items-start ">`
this:
 ` <div className="w-48 h-full flex flex-col  items-center  text-base font-normal sm:<w-auto item-start flex-row>">`

**Comment by wongjn** [comment]
[Here's a response to a similar suggestion](https://github.com/tailwindlabs/tailwindcss/discussions/9053#discussioncomment-3350470):

> We get this question so many times, so I would like to redirect you here for more information: https://twitter.com/adamwathan/status/1461519820411789314

**Reply by detoner777** [reply]
Thank you for sharing!
